main.rb

```diff
@@ -0,0 +1,49 @@
+require_relative 'App'
```

Comment:
- [ ] kindly change `App` to `app` as you require relative

Reply:
Thank you, this has been fixed now